pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -60,6 +65,12 @@ message MessageMetadata {
 	//optional sfixed64 checksum = 10;
 	// differentiate single and batch message metadata
 	optional int32 num_messages_in_batch = 11 [default = 1];
+	// Comman separated encryption key name(s) used to encrypt data key
```

Comment:
If `encryption_keys` is effectively a map, the comment should no mention "comma-separated"